08LessonEight/in-class-project/homework.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Blog Page</title>
+    <link type="text/css" rel="stylesheet" href="css/homework-style.css" />
+</head>
+
+<body>
+    <header>
+        <section>
+            <p id="recentblog">Recent From Blog</p>
+            <p>
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s..
+            </p>
+        </section>
+    </header>
+    <br />
+</body>
+<section class="mainframe clearfix">
+    <div class="leftside">
+        <div class="keyboardbox">
+            <img src="img/keyboard.jpg" />
+        </div>
+        <div>
+            <p class="title">Morbi eu metus ut libero laoreet vulputate</p>
+            <p>
+                It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+            </p>
+            <a href="#" class="clearfix">Read More</a>
+        </div>
+    </div>
+    <div class="rightside clearfix">
```

Comment:
Yes, I've divided the left and right side which are both floating. 

    On Saturday, August 27, 2016 9:49 PM, Erik Zambrano <notifications@github.com> wrote:
 

 In 08LessonEight/in-class-project/homework.html:> +    <br />
> +</body>
> +<section class="mainframe clearfix">
> +    <div class="leftside">
> +        <div class="keyboardbox">
> +            <img src="img/keyboard.jpg" />
> +        </div>
> +        <div>
> +            <p class="title">Morbi eu metus ut libero laoreet vulputate</p>
> +            <p>
> +                It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
> +            </p>
> +            <a href="#" class="clearfix">Read More</a>
> +        </div>
> +    </div>
> +    <div class="rightside clearfix">
Are all the child elements of this div floating?—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.